### ISSUE TITLE
Odoov11E : list of input values popup staying in form / blocking formview.

### ISSUE THREAD

**Issue by dipsshah42** [odoo/odoo#23056]
Impacted versions:
	> Odoo V11E

This is showing as generic issue in all the places of odoo.

Steps to reproduce:
	> Open Members menu,
	> Create > Click on Company field ,
	> So list of Partners will appear ,
	> Now without clicking anywhere just scroll in formview.

Current behavior:
	> list of items small popup blocking the other parts of formview also.
	> and this is generic for all input box popping up on fields. 

Expected behavior:
	> Small input popup should be disappear when scrolling a page/formview.

Video/Screenshot link (optional):

Video > 	https://youtu.be/uUcdHfqXNfc
Screenshot > 	https://www.awesomescreenshot.com/image/3172515/654137bfe008593e8e482d21b65b6e91

Thanks.

**Comment by Yenthe666** [comment]
@dipsshah42 I honestly don't consider this a bug, perhaps a minor usability / friendly thing but hmm, there is no perfect way to make this ideal for anyone.
@mart-e the final call is up to you for this one.

**Comment by dipsshah42** [comment]
@Yenthe666 Agreed with you but this is on all places so, what if we will make that input box disappear when scrollling the other parts of page/formview.(from .js side we need to track the scrolling and need to do something so that box will disappear) then it will increase the usability at better level.what you think ?
Thanks!

**Comment by mart-e** [comment]
Hmm on which browser are you? I did not get the problem on firefox.
Are you checking on an up to date 11.0 ?

**Comment by Yenthe666** [comment]
@dipsshah42 please provide us with extra info.

**Comment by dipsshah42** [comment]
@mart-e  and @Yenthe666 ,
This is happening in almost all browser, i'm facing with ,

1.**Google Chrome** Version 63.0.3239.108 (Official Build) (64-bit)
2.**Chromium** Version 64.0.3282.140 (Official Build) Built on Ubuntu , running on Ubuntu 16.04 (64-bit)

Do you guys need any other details please let me know ? as i said this is generic and happening in mostly all the formview's selection and relation kind of fields.

Thanks !

**Comment by dipsshah42** [comment]
Yes, i tried on v11 and v12 Community versions too.its still there @pedrobaeza .

**Comment by Yenthe666** [comment]
Test on Chrome `71.0.3578.98` but cannot reproduce it there
